teamB/jungJaeUk/app.js

```diff
@@ -49,6 +95,83 @@ app.post('/signup', async (req, res) => {
   );
   res.status(201).json({ "message" : "userCreated" });
 });
+//[POST] post add
+app.post('/posts/add', async (req, res) => {
+  const { title, content, user_id } = req.body;
+
+  await mysqlDataSource.query(
+    `
+    INSERT INTO posts (
+      title,
+      content,
+      user_id
+    ) VALUES ( ?, ?, ? );
+    `,
+    [ title, content, user_id ]
+  );
+
+  res.status(201).json({ "message" : "postCreated" });
+});
+//[POST] post update 
+app.post('/posts/update', async (req, res) => {
+  const { user_id, post_id, content } = req.body;
+
+  await Promise.all([
+    await mysqlDataSource.query(
+      `
+      UPDATE posts
+      SET content="${content}"
+      WHERE user_id=${user_id} AND id=${post_id};
+      `
+    ),
+    mysqlDataSource.query(
+      `
+      SELECT
+      users.id as userId,
+      users.name as userName,
+      posts.id as postingId,
+      posts.title as postingTitle,
+      posts.content as postingContent
+      FROM users INNER JOIN posts
+      ON users.id = posts.user_id AND users.id=${user_id} AND posts.id=${post_id};
+      `
+    )
```

Comment:
- SQL 포맷팅을 지켜서 수정해주세요 !

```suggestion
    await mysqlDataSource.query(`
      UPDATE posts
      SET content="${content}"
      WHERE user_id=${user_id} AND id=${post_id};
      `),
    mysqlDataSource.query(`
      SELECT
        users.id as userId,
        users.name as userName,
        posts.id as postingId,
        posts.title as postingTitle,
        posts.content as postingContent
      FROM users 
      INNER JOIN posts ON users.id = posts.user_id 
      AND users.id=${user_id} AND posts.id=${post_id};`
    )
```

Reply:
수정하겠습니다.